core/assertion.ts

```diff
@@ -27,6 +27,16 @@ export class Assertion {
     return this;
   }
 
+  public tags(value: string | string[]) {
+    const newTags = typeof value === "string" ? [value] : value;
+    newTags.forEach(d => {
```

Comment:
nit: let's rename `d` to `t`

(this is repeated a couple of places in this PR)

Reply:
Done